src/sharding/ShardingManager.js

```diff
@@ -43,20 +45,43 @@ class ShardingManager extends EventEmitter {
     const stats = fs.statSync(this.file);
     if (!stats.isFile()) throw new Error('CLIENT_INVALID_OPTION', 'File', 'a file');
 
+    this._maxRecommended = await Util.fetchRecommendedShards(this.token);
+
+    /**
+     * First shard this sharding manager will spawn
+     * @type {number}
+     */
+    this.startShard = options.startShard;
+    if (typeof this.startShard !== 'number' || isNan(this.startShard)) {
+      throw new TypeError('CLIENT_INVALID_OPTION', 'Start shard', 'a number.');
+    }
+    if (this.startShard < 1) throw new RangeError('CLIENT_INVALID_OPTION', 'Start shard', 'at least 1.');
+    if (this.startShard > this._maxRecommended) {
+      throw new RangeError('CLIENT_INVALID_OPTION', 'Start shard', `no more than ${this._maxRecommended} (recommended amount)`);
```

Comment:
I disagree with forcing recommended shard count or less.  People should be able to spawn however many shards they want to suit their needs.